src/__tests__/timer-test.tsx

```diff
@@ -0,0 +1,142 @@
+import * as React from 'react';
+import {render, screen} from '@testing-library/react';
+import {TextTimer, ThemeContextProvider, Timer} from '..';
+import {makeTheme} from './test-utils';
+import {act} from 'react-dom/test-utils';
+
+const SECOND = 1000;
+const DAY = SECOND * 60 * 60 * 24;
+
+test('Timer', async () => {
+    jest.useFakeTimers();
+
+    render(
+        <ThemeContextProvider theme={makeTheme()}>
+            <Timer endTimestamp={Date.now() + 60 * SECOND} />
+        </ThemeContextProvider>
+    );
+
+    await screen.findByRole('timer');
+
+    await screen.findByText('0 horas, 1 minuto y 0 segundos');
+
+    act(() => jest.advanceTimersByTime(SECOND));
+    await screen.findByText('0 horas, 0 minutos y 59 segundos');
+
+    act(() => jest.advanceTimersByTime(SECOND * 29));
+    await screen.findByText('0 horas, 0 minutos y 30 segundos');
+
+    act(() => jest.runAllTimers());
+    await screen.findByText('0 horas, 0 minutos y 0 segundos');
+});
+
+test('Timer - timestamp from the past', async () => {
+    jest.useFakeTimers();
+
+    render(
+        <ThemeContextProvider theme={makeTheme()}>
+            <Timer endTimestamp={Date.now() - SECOND} />
+        </ThemeContextProvider>
+    );
+
+    await screen.findByText('0 horas, 0 minutos y 0 segundos');
+
+    act(() => jest.runAllTimers());
+    await screen.findByText('0 horas, 0 minutos y 0 segundos');
+});
+
+test('Timer - from minutes to days', async () => {
+    jest.useFakeTimers();
+
+    render(
+        <ThemeContextProvider theme={makeTheme()}>
+            <Timer endTimestamp={Date.now() + 61 * SECOND} minTimeUnit="minutes" maxTimeUnit="days" />
+        </ThemeContextProvider>
+    );
+
+    await screen.findByText('0 días, 0 horas y 1 minuto');
+
+    act(() => jest.advanceTimersByTime(SECOND));
+    await screen.findByText('0 días, 0 horas y 1 minuto');
+
+    act(() => jest.advanceTimersByTime(SECOND));
+    await screen.findByText('0 días, 0 horas y 0 minutos');
+
+    act(() => jest.runAllTimers());
+    await screen.findByText('0 días, 0 horas y 0 minutos');
+});
+
+test('Timer - renders only minimum unit if minTimeUnit is bigger than maxTimeUnit', async () => {
```

Comment:
I miss a test for `<Timer endTimestamp={Date.now() + DAY} maxTimeUnit="seconds" />` finding text `"86400 segundos"`

Reply:
added